packages/design-system-react/figma.config.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Same as React Native config but allows us to have multiple dropdowns and specifc import paths